blog/2023-05-24-spec-0-6-0.md

```diff
@@ -0,0 +1,49 @@
+---
+slug: 'spec-0.6.0'
+title: 'Specification 0.6.0'
+date: 2023-05-24
+authors: [beeme1mr]
+description: 'Announcing version 0.6.0 of the OpenFeature specification'
+keywords: [release, '0.6.0', openFeature, specification, spec]
+tags: [spec, specification, sdk]
+draft: false
+---
+
+We are excited to announce the availability of version 0.6.0 of the OpenFeature spec. This release brings many significant additions that benefit both provider developers and end users, while also establishing a strong foundation for first-class client-side support in OpenFeature.
```

Comment:
```suggestion
We are excited to announce the availability of version 0.6.0 of the OpenFeature spec. This release brings many significant additions that benefit both provider developers and application authors, while also establishing a strong foundation for first-class client-side support in OpenFeature.
```

Reply:
Do you mean actual end-users, or people writing applications with OpenFeature? I suspect the latter.